File: despatch.WEB/app/src/swagger/models/user-phone-numbers-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { PhoneNumberRequest } from './phone-number-request';
import { UserRequest } from './user-request';
export interface UserPhoneNumbersRequest extends BaseEntityRequest{
  phoneNumber?: PhoneNumberRequest;
  phoneNumberId?: string;
  user?: UserRequest;
  userId?: string;
}

File: despatch.Domain/Models/Default/Request/UserPhoneNumbersRequest.cs
namespace despatch.Domain.Models.Default.Request
{
    public class UserPhoneNumbersRequest : BaseEntityRequest
    {
        public string UserId { get; set; }
        public string PhoneNumberId { get; set; }

        public virtual UserRequest User { get; set; }
        public virtual PhoneNumberRequest PhoneNumber { get; set; }
    }
}
